src/main/grammars/RustParser.bnf

```diff
@@ -113,17 +113,32 @@ File ::= [ SHEBANG_LINE ] InnerAttr* Items
 // Attributes
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-fake Attr ::= MetaItem
+fake Attr ::= MetaItem {
+  extends = "org.rust.lang.core.psi.ext.RsStubbedElementImpl<?>"
+  stubClass = "org.rust.lang.core.stubs.RsPlaceholderStub"
+}
```

Comment:
You want to save all `RsMetaItem` elements (identifier and literal value) into stub and use them, don't you?